synapse/models/inet.py

```diff
@@ -940,12 +940,27 @@ def getBaseModels():
                     ('email', {'ptype': 'inet:email'}),
                     ('phone', {'ptype': 'tel:phone'}),
                     ('signup', {'ptype': 'time', 'doc': 'The time the web account was registered'}),
-                    ('signup:ipv4', {'ptype': 'inet:ipv4', 'doc': 'The original ipv4 address used to sign up for the account'}),
+                    ('signup:ipv4', {'ptype': 'inet:ipv4',
+                                     'doc': 'The original ipv4 address used to sign up for the account'}),
                     ('passwd', {'ptype': 'inet:passwd', 'doc': 'The current passwd for the web account'}),
                     ('seen:min', {'ptype': 'time:min'}),
                     ('seen:max', {'ptype': 'time:max'}),
                 ]),
 
+                ('inet:web:chprofile', {}, [
+                    ('acct', {'ptype': 'inet:web:acct', 'doc': 'Account associated with the change', 'ro': 1,
+                              'req': 1}),
+                    ('acct:site', {'ptype': 'inet:fqdn', 'doc': 'Site the user account is at', 'ro': 1, }),
+                    ('acct:user', {'ptype': 'inet:user', 'doc': 'User account name', 'ro': 1, }),
+                    ('ipv4', {'ptype': 'inet:ipv4', 'doc': 'Source IPv4 Address the account was done from', }),
+                    ('ipv6', {'ptype': 'inet:ipv6', 'doc': 'Source IPv6 Address the account was done from', }),
+                    ('time', {'ptype': 'time', 'doc': 'When the account change was done', }),
+                    ('pv', {'ptype': 'propvalu', 'doc': 'The value changed in the account', 'ro': 1, 'req': 1}),
+                    ('pv:prop', {'ptype': 'str', 'doc': 'Property which changed', 'ro': 1}),
+                    ('pv:strval', {'ptype': 'str', 'doc': 'System normed string value', 'ro': 1}),
```

Comment:
Suggest 'The normed value of the property (specified by pv), if the property is a string.'
Suggest 'The normed value of the property (specified by pv), if the property is an integer.'
Keeps consistent with similar docstrings in files.py.

Reply:
fixed